run.py

```diff
@@ -132,34 +83,46 @@ def check_inputs(context):
     apply_to_files = []
     index_list = []
 
+    # Capture all the inputs from the gear context
     image1_path = context.get_input_path('image_1')
     image2_path = context.get_input_path('image_2')
     config_path = context.get_input_path('config_file')
     apply_to_a = context.get_input_path('apply_to_1')
     apply_to_b = context.get_input_path('apply_to_2')
     acq_par = context.get_input_path('acquisition_parameters')
 
+    # If image_1 is 4D, we will apply topup correction to the entire series after running topup
```

Comment:
since you are zipping apply_to_files and index_list afterwards, I would have just zip them as you are building the array here, younameit.append((image1_path, '1')), etc...